cmd/root.go

```diff
@@ -25,6 +27,7 @@ const (
 	argRootSkipVerify = "skip-verify"
 	argRootToken      = "token"
 	argRootVerbose    = "verbose"
+	argRootGenerate   = "generate"
```

Comment:
Yes, this is much better!

However, I vote for renaming this to `autocomplete-generate`. I think `generate` by itself is too generic, and sounds too much like it has something to do with Artifacts. Not to mention that we might want to use this keyword for something more common later on. `autocomplete-generate` will probably only be run once, or a few times at most.

Reply:
How about `generate-autocomplete` ? :)
Rolls off the tongue more I feel

Reply:
:+1: